opflexagent/utils/bridge_managers/vpp_manager.py

```diff
@@ -33,6 +34,7 @@ def __init__(self):
     def initialize(self, host, conf, agent_state):
         self.int_br_device_count = 0
         vpp_config = conf.VPP
+        agent_state['agent_type'] = ofcst.AGENT_TYPE_OPFLEX_OVS
```

Comment:
this should be AGENT_TYPE_OPFLEX_VPP ?

Reply:
Definitely :-(. Will respin, and thanks for catching!